admin/src/amp-img-widget.js

```diff
@@ -73,7 +73,7 @@ CMS.registerEditorComponent({
     {name: 'attrlink', label: 'Figure attribution link', widget: 'string'}
   ],
   // Pattern to identify a block as being an instance of this component
-  pattern: /^{{ figure.html (.*?) }}$/,
+  pattern: /^{% include figure.html (.*?) %}$/,
```

Comment:
this is accepted! now i can test if it is correctly interpeted...

Reply:
It is interpreted in the right way, i'm gonna straight to implement all the code.